gno.land/pkg/integration/testdata/improved-coins.txtar

```diff
@@ -0,0 +1,94 @@
+gnoland start
+
+gnokey maketx addpkg -pkgdir $WORK -pkgpath gno.land/r/demo/coins -gas-fee 1000000ugnot -gas-wanted 20000000 -broadcast -chainid=tendermint_test test1
```

Comment:
I think you can use `loadpkg` here

Reply:
Switched it out: [16109c4](https://github.com/gnolang/gno/pull/2104/commits/16109c4d5cfcbb59f09128bb48fd6de78a0e2465)